keystore/filesystem/server_keystore.go

```diff
@@ -387,12 +390,55 @@ func (store *KeyStore) GetPublicKeyFilePath(filename string) string {
 	return fmt.Sprintf("%s%s%s", store.publicKeyDirectory, string(os.PathSeparator), filename)
 }
 
+// use key started with "." (dot) because it's invalid character for clientID that generally stored in cache and
+// it will not intersect with other keys
+const cacheKeySuffix = ".historical."
```

Comment:
Based on how you use it, it's more like a `cacheKeyPrefix` :thinking:

Reply:
yeah, renamed